core/tests/behavior/utils.rs

```diff
@@ -83,13 +83,18 @@ pub fn init_service<B: Builder>() -> Option<Operator> {
     };
 
     let _guard = RUNTIME.enter();
```

Comment:
We can remove this line when we don't need blocking layer.

Reply:
That's what I want to ask. 😆

Reply:
DONE